### PR TITLE
Test docker build

### PR DESCRIPTION
**Pull request by afrittoli**
Fixes #ISSUE_NUMBER

cc @seemethere @malfet @pytorch/pytorch-dev-infra

### CODE REVIEW

File: .github/workflows/_linux-build.yml [hunk 1]
```diff
@@ -136,7 +136,7 @@ jobs:
 
       - name: Calculate docker image
         id: calculate-docker-image
-        uses: pytorch/test-infra/.github/actions/calculate-docker-image@main
+        uses: pytorch/test-infra/.github/actions/calculate-docker-image@test-calculate-docker-image
         if: inputs.build-environment != 'linux-s390x-binary-manywheel'
         with:
           docker-image-name: ${{ inputs.docker-image-name }}
```
Comment (atalman) [inline .github/workflows/_linux-build.yml:142]:
```suggestion
          docker-image-name: ${{ inputs.docker-image-name }}
          always-rebuild: true
```

File: .ci/docker/common/install_conda.sh [hunk 2]
```diff
@@ -84,8 +85,9 @@ if [ -n "$ANACONDA_PYTHON_VERSION" ]; then
 
   # Magma package names are concatenation of CUDA major and minor ignoring revision
   # I.e. magma-cuda102 package corresponds to CUDA_VERSION=10.2 and CUDA_VERSION=10.2.89
+  # Magma is installed from a tarball in the ossci-linux bucket into the conda env
   if [ -n "$CUDA_VERSION" ]; then
-    conda_install magma-cuda$(TMP=${CUDA_VERSION/./};echo ${TMP%.*[0-9]}) -c pytorch
+    ${SCRIPT_FOLDER}/install_magma_conda.sh $(cut -f1-2 -d'.' <<< ${CUDA_VERSION}) ${ANACONDA_PYTHON_VERSION}
```
Comment (atalman) [inline .ci/docker/common/install_conda.sh:90]:
```suggestion
    conda_install magma-cuda$(TMP=${CUDA_VERSION/./};echo ${TMP%.*[0-9]}) -c pytorch
```

File: .ci/docker/ubuntu/Dockerfile [hunk 3]
```diff
@@ -36,7 +36,8 @@ ENV DOCS=$DOCS
 COPY requirements-ci.txt requirements-docs.txt /opt/conda/
 COPY ./common/install_conda.sh install_conda.sh
 COPY ./common/common_utils.sh common_utils.sh
-RUN bash ./install_conda.sh && rm install_conda.sh common_utils.sh /opt/conda/requirements-ci.txt /opt/conda/requirements-docs.txt
+COPY ./common/install_magma_conda.sh install_magma_conda.sh
+RUN bash ./install_conda.sh && rm install_conda.sh install_magma_conda.sh common_utils.sh /opt/conda/requirements-ci.txt /opt/conda/requirements-docs.txt
```
Comment (atalman) [inline .ci/docker/ubuntu/Dockerfile:40]:
```suggestion
RUN bash ./install_conda.sh && rm install_conda.sh common_utils.sh /opt/conda/requirements-ci.txt /opt/conda/requirements-docs.txt
```

**Review by malfet — CHANGES_REQUESTED** [review]
As it doesn't not reference any issue, it would be nice to provide a better description on what this PR is trying to achieve, as `Test docker build` is a big vague

File: .github/workflows/_linux-build.yml [hunk 4]
```diff
@@ -136,10 +136,11 @@ jobs:
 
       - name: Calculate docker image
         id: calculate-docker-image
-        uses: pytorch/test-infra/.github/actions/calculate-docker-image@main
+        uses: pytorch/test-infra/.github/actions/calculate-docker-image@test-calculate-docker-image
```
Comment (malfet) [inline .github/workflows/_linux-build.yml:139]: This is a temp change, right?

Comment (afrittoli) [reply .github/workflows/_linux-build.yml:139]: Yes, just for testing purposes, not meant to be reviewed or merged 🙏

**Pull request closed without merging**